ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -176,6 +176,21 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-nutanix-ovn
+  cluster: build01
```

Comment:
Do we need to specifically run it off of `build01`?

Reply:
I think so, that's where we have a VPN for nutanix prism central CI server. @adiantum can you confirm?

Reply:
@damdo In the best of my knowledge VPN operator required to access CI cluster present only on `build01` cluster.

Reply:
Gotcha, thanks for explaining!